cassandra-framework/src/main/java/io/mesosphere/mesos/frameworks/cassandra/Main.java

```diff
@@ -62,8 +62,9 @@ private static int _main() throws UnknownHostException {
         final long      resourceMemoryMegabytes = Long.parseLong(       Env.option("CASSANDRA_RESOURCE_MEM_MB").or("2048"));
         final long      resourceDiskMegabytes   = Long.parseLong(       Env.option("CASSANDRA_RESOURCE_DISK_MB").or("2048"));
         final long      healthCheckIntervalSec  = Long.parseLong(       Env.option("CASSANDRA_HEALTH_CHECK_INTERVAL_SECONDS").or("60"));
+        final String    cassandraVersion        =                       Env.option("CASSANDRA_VERSION").or("2.1.2");
```

Comment:
Lets not read this variable from the env until we know that we're going to allow it to be changed.  It's okay to pass it to the scheduler, but lets use a constant for the value.